docs/writing-docs/mdx.md

```diff
@@ -254,36 +285,77 @@ import Changelog from '../CHANGELOG.md';
 
 -->
 
-## Linking to other stories and pages
+## Troubleshooting
 
-When writing MDX, you may want to provide links to other stories or documentation pages and sections. You can use the `path` query string.
+### The migration seems flaky and keeps failing
 
-Considering a story with ID `some--id`, this redirects to the **Docs** tab of the story:
+By default, running the [migration command](#automigration) will try and migrate all existing MDX files in your project according to the MDX 2 specification. However, this might not always be possible, and you might run into issues during the migration. To help you troubleshoot those issues, we've prepared some recommendations that might help you.
 
-```md
-[Go to specific documentation page](?path=/docs/some--id)
+Start by running the following command inside your project directory:
+
+```shell
+npx @hipster/mdx2-issue-checker
 ```
 
-This redirects to the **Canvas** tab of the story:
+<div class="aside">
+💡 Depending on the volume, you may be required to run the command multiple times to fix all the issues.
+</div>
 
-```md
-[Go to specific story canvas](?path=/story/some--id)
+When it finishes, it will output the list of files causing issues. You can then use this information to fix the problems manually.
+
+Additionally, if you're working with VSCode, you can add the [MDX extension](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx) and enable MDX experimental support by adding the following to your user settings:
+
+```json
+{
+  "workbench.startupEditor": "newUntitledFile",
+  "workbench.iconTheme": "vscode-icons",
+  "editor.minimap.enabled": false,
+  "editor.formatOnSave": true,
+  "editor.formatOnType": true,
+  "editor.formatOnPaste": true,
+  "editor.inlineSuggest.enabled": true,
+  "editor.bracketPairColorization.enabled": true,
+  "editor.guides.bracketPairs": "active",
+  "editor.stickyScroll.enabled": true,
+  "mdx.experimentalLanguageServer": true
+}
```

Comment:
It's only really the last setting that's relevant here, right?
```suggestion
{
  "mdx.experimentalLanguageServer": true
}
```

Reply:
I'm good with the suggestion and have already applied it. I left it so that I was able to give some context on what the actual structure be.